bin/ch/ch.cpp

```diff
@@ -129,8 +128,8 @@ void __stdcall PrintChakraCoreVersion()
                 // Doesn't matter if you are on 32 bit or 64 bit,
                 // DWORD is always 32 bits, so first two revision numbers
                 // come from dwFileVersionMS, last two come from dwFileVersionLS
-                printf("%s version %d.%d.%d.%d\n",
-                    chakraDllName,
+                wprintf(_u("%s version %d.%d.%d.%d\n"),
+                    GetChakraDllNameW(),
```

Comment:
Looks like %S does what we'd expect in PAL (behaves the same on Linux and Windows). On Linux, it does seem like the implementation wprintf flips the meanings of those formatters.

In any case for this bit of output it doesn't matter because it's Windows-only.